bokehjs/src/coffee/models/glyphs/quad.coffee

```diff
@@ -25,16 +25,10 @@ class QuadView extends Glyph.View
 
     for i in [0...left.length]
       l = left[i]
-      if isNaN(l) or not isFinite(l)
```

Comment:
the original code checked for `NaN`/`isFinite` on four variables independently. But those properties are viral: if any of a, b, c, d satisfy them, then their sum also satisfies them.